src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/AttributesAccordion.tsx

```diff
@@ -11,26 +11,58 @@ import {
   AccordionItemPermanentBlock
 } from 'src/shared/components/accordion';
 
-import { getAttributesAccordionExpandedPanel } from '../../../state/attributes/attributesSelector';
+import {
+  getAttributesAccordionExpandedPanels,
+  getSelectedAttributes
+} from '../../../state/attributes/attributesSelector';
+import BadgedButton from 'src/shared/components/badged-button/BadgedButton';
 import {
   setAttributesAccordionExpandedPanel,
   fetchAttributes,
   resetSelectedAttributes
 } from '../../../state/attributes/attributesActions';
-
 import { Orthologues } from './sections';
-import customDownloadStorageService from 'src/content/app/custom-download/services/custom-download-storage-service';
-
+import { setShowExampleData } from 'src/content/app/custom-download/state/customDownloadActions';
 import ImageButton, {
   ImageButtonStatus
 } from 'src/shared/components/image-button/ImageButton';
 import { ReactComponent as ResetIcon } from 'static/img/shared/trash.svg';
+import JSONValue from 'src/shared/types/JSON';
+import AttributesAccordionSection from 'src/content/app/custom-download/containers/content/attributes-accordion/sections/AttributesAccordionSection';
 import styles from './AttributesAccordion.scss';
 
-import AttributesAccordionSection from 'src/content/app/custom-download/containers/content/attributes-accordion/sections/AttributesAccordionSection';
+type StateProps = {
+  expandedPanels: string[];
+  selectedAttributes: JSONValue;
+};
+
+type DispatchProps = {
+  setAttributesAccordionExpandedPanel: (expandedPanels: string[]) => void;
+  fetchAttributes: () => void;
+  resetSelectedAttributes: () => void;
+  setShowExampleData: (showExampleData: boolean) => void;
+};
 
 type Props = StateProps & DispatchProps;
 
+const getTotalSelectedAttributes = (
+  attributes: JSONValue,
+  totalSelectedAttributes = 0
+) => {
+  Object.keys(attributes).forEach((key) => {
+    if (typeof attributes[key] === 'boolean' && attributes[key] === true) {
+      totalSelectedAttributes++;
+    } else if (typeof attributes[key] === 'object') {
```

Comment:
Just curious here: can `attributes[key]` be an array? If yes, it will be captured by this if statement. Will it be ok?

Reply:
No, it cannot be an array at the moment.